Tasks/AzureTestPlanV0/automatedTests.ts

```diff
@@ -1,28 +1,38 @@
 import tl = require('azure-pipelines-task-lib/task');
-import { testInvoker } from './automatedTestInvoker'
-import { TestPlanData } from './testPlanData'
-import { publishAutomatedTestResult } from './publishAutomatedTests'
+import { testInvoker } from './automatedTestInvoker';
+import { TestPlanData } from './testPlanData';
+import { publishAutomatedTestResult } from './publishAutomatedTests';
 
+export async function automatedTestsFlow(testPlanInfo: TestPlanData, testSelectorInput: string): Promise<number> {
+  let listOfTestsToBeExecuted: string[] = testPlanInfo.listOfFQNOfTestCases;
+  let testInvokerStatusCode = 0;
 
-export async function automatedTestsFlow(testPlanInfo: TestPlanData, testSelectorInput: string) {
+  if (listOfTestsToBeExecuted !== null && listOfTestsToBeExecuted !== undefined && listOfTestsToBeExecuted.length > 0) {
+    tl.debug('Invoking test execution for tests: ' + listOfTestsToBeExecuted);
 
-    let listOfTestsToBeExecuted: string[] = testPlanInfo.listOfFQNOfTestCases;
-
-    console.log(tl.loc('automatedTestTriggered'));
-
-    if (listOfTestsToBeExecuted !== null && listOfTestsToBeExecuted !== undefined && listOfTestsToBeExecuted.length > 0) {
-        tl.debug("Invoking test execution for tests: " + listOfTestsToBeExecuted);
-        await testInvoker(listOfTestsToBeExecuted);
-        publishAutomatedTestResult(JSON.stringify(testPlanInfo.listOfAutomatedTestPoints));
+    try {
+      testInvokerStatusCode = await testInvoker(listOfTestsToBeExecuted);
+    } catch (err) {
+      tl.debug(`Unable to invoke automated test execution. Err:( ${err} )`);
```

Comment:
Is this expected, we are catching the err but not returning  1 or setting the testInvokerStatusCode?

Reply:
Updated to have the testInvokerStatusCode as 1